tests/gumjs/script.c

```diff
@@ -3465,9 +3512,9 @@ invoke_target_function_int_worker (gpointer data)
 {
   GumInvokeTargetContext * ctx = (GumInvokeTargetContext *) data;
 
-  ctx->started = TRUE;
+  ctx->started++;
   target_function_int (42);
-  ctx->finished = TRUE;
+  ctx->finished++;
```

Comment:
I think we should switch these two operations to [g_atomic_int_inc()](https://developer.gnome.org/glib/stable/glib-Atomic-Operations.html#g-atomic-int-inc) now that there's a theoretical chance of two threads stepping on each other.